docs/swagger.yaml

```diff
@@ -6861,6 +6876,39 @@ paths:
       tags:
         - user
     parameters: []
+  /user/{api_version}/account/login_session/:
+    get:
+      operationId: user_account_login_session_list
+      description: HTTP end-points for logging in users.
+      parameters: []
+      responses:
+        '200':
+          description: ''
+      tags:
+        - user
+    post:
+      operationId: user_account_login_session_create
+      summary: Log in a user.
+      description: |-
+        See `login_user` for details.
+
+        Example Usage:
+
+            POST /api/user/v1/login_session
+            with POST params `email`, `password`.
+
+            200 {'success': true}
+      parameters: []
+      responses:
+        '201':
+          description: ''
+      tags:
+        - user
+    parameters:
+      - name: api_version
+        in: path
+        required: true
+        type: string
```

Comment:
It was auto-generated by `make swagger` as you guessed. 
I don't think api_version can be parameterised here since there is only one version of the API.